### ISSUE TITLE
Support alternative disc formats

### ISSUE THREAD

**Issue by JaCzekanski** [JaCzekanski/Avocado#41]
As mentioned in #28, support for different disc dump formats is a nice feature to have for end-user.

As for now Avocado supports:
- [x] single file .bin/.img/.iso (2352) images
- [x] multitrack .cue image with .bin tracks
- [x] MAME .chd disc file format with FLAC audio and LZMA compressed data
- [x] .ecm files
- [x] PSExe files (.exe, .psexe)
- [x] PSF music (.psf, .minipsf)

Nice to have:
- [ ] .cue with WAV/MP3 tracks
- [ ] PS Classics .pbp files download from Playstation Store

If you have any new file format suggestions feel free to comment on this issue.

**Comment by shikulja** [comment]
add in the future 7z-zip multi-images possible?

in 7z it’s very convenient to pack almost identical images for example (pal-ntsc, different versions, and translations) especially when they have similar external audio tracks.

while other emulators do not have this advantage

![Скриншот 27-08-2019 025833](https://user-images.githubusercontent.com/3386359/63731041-a3202600-c876-11e9-919d-de170f9dc939.png)

you can use to create and verify:
https://github.com/johndimi/cdcrush.net

**Comment by JaCzekanski** [comment]
If 7z format allows for random access and seeking without fully unpacking its contents I think it's doable. I'm sure that it can be done for simpler formats like .zip (other emulators has done it).

**Comment by shikulja** [comment]
+ .mdf / .mds is not the right format in which images still exist.

**Comment by JaCzekanski** [comment]
@shikulja I don't think this format has any popularity in emulation scene - correct me if I'm wrong. 

I'll take it into considerations.

**Comment by shikulja** [comment]
> @shikulja I don't think this format has any popularity in emulation scene - correct me if I'm wrong.
> 
> I'll take it into considerations.

no has, but sometimes in this format, images are found.
They can be converted. But it’s better not to touch such images; the quality of dumps is sometimes difficult to judge.

**Comment by JaegerFox** [comment]
Support for zipped/7z files with zstandard algorithm(?
is a real-time compression algorithm, providing high compression ratios. It offers a very wide range of compression / speed trade-off, while being backed by a very fast decoder
and recently it can be compressed with the zip format using 7zip-zstd
